prime-router/metadata/fhir_mapping/hl7/segments/ORC/ServiceRequest.yml

```diff
@@ -0,0 +1,368 @@
+resourceType: ServiceRequest
+
+# Not Implementing
+# ORC.7 deprecated in the NIST spec
+# ORC.26 deprecated in the NIST spec
+# ORC.32 not mapped, not in NIST spec
+# ORC.34 not mapped, not in NIST spec
+# OBR.5 deprecated in NIST spec
+# OBR.6 deprecated in NIST spec
+# OBR.30 deprecated in NIST spec
+# OBR.37 deprecated in NIST spec
+# OBR.38 deprecated in NIST spec
+# OBR.40 deprecated in NIST spec
+# OBR.41 deprecated in NIST spec
+# OBR.42 deprecated in NIST spec
+# OBR.43 deprecated in NIST spec
+# OBR.54 not mapped, not in NIST
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+status:
+  type: SERVICE_REQUEST_STATUS
+  default: "unknown"
+  valueOf: ORC.5 | ORC.1
+  expressionType: HL7Spec
+
+intent:
+  expressionType: nested
+  vars:
+    obr11: STRING, OBR.11
+  expressions:
+    - condition: $obr11 NOT_EQUALS G || $obr11 NOT_EQUALS A
+      type: STRING
+      value: "order"
+    - condition: $obr11 EQUALS A
+      type: STRING
+      value: "#add-on#"
+    - condition: $obr11 EQUALS G
+      type: STRING
+      value: "reflex-order"
+_intent:
+  expressionType: nested
+  expressionsMap:
+    extension:
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: obr-11-specimen-action-code
+        valueId:
+          type: STRING
+          valueOf: OBR.11
+          expressionType: HL7Spec
+
```

Comment:
Why is this extension added here? I'm not seeing it in the mapping and `obr-11-specimen-action-code` is also in the `obr-observation-request` list.

Reply:
Oh sorry, I see, yea typo.  I'll remove the one in the extension